app/konnect/runtime-manager/gateway-config.md

```diff
@@ -113,3 +113,15 @@ ClientHello to look up the associated certificate object.
 
 See the [SNI object API reference](/gateway/latest/admin-api/#sni-object)
 for all configuration options.
+
+
+### Vaults
+
+Add authentication to a service or route with an access token and secret token. Credential tokens are stored securely using Vaults. Credential life-cyles can be managed through {{site.konnect_short_name}}. 
```

Comment:
The first sentence kind of reads as more feature description-y? IDK how to describe it. Like I might expect something more like "You can use Vaults to...." or "When you add authentication...".